nexus/users/migrations/0004_auto_20180925_1334.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.0.5 on 2018-09-24 14:44
+# Generated by Django 2.0.5 on 2018-09-25 13:34
 
```

Comment:
This is a breaking change. Your current migration number should be changed. Not the one which is already deployed.